### ISSUE TITLE
Not immediate way to know if an item has expired using the toolbar

### ISSUE THREAD

**Issue by hvelarde** [plone/Products.CMFPlone#1577]
Information about the expiration of a piece of content was explicit on the document byline; this is not the case with the toolbar:

![selection_070](https://cloud.githubusercontent.com/assets/341393/15051088/5abf3e44-12cd-11e6-9060-aa3f3d72b876.png)

**Comment by jensens** [comment]
imo this should be shown together with the state. First idea: strike trough style on the state name itself?

**Comment by davilima6** [comment]
I agree it's closely related to State, I'm afraid too close even since Expired is a state but not a Workflow state. Is it a Publish state? Or more like a Visibility (for content listings) state? We could add a separator/header to the current Workflow State dropdown menu to make clear when current context will be effective and expired, if ever.

**Comment by djay** [comment]
@plone/ux-team

**Comment by tkimnguyen** [comment]
Because it's not really state, it needs a separate indicator on the toolbar. Strike-through style implies too much, I think. An actual word like "Expired" would be more immediately understood.

**Comment by jensens** [comment]
I do not agree. For the editor its semantically a state. Technically not.


Am 06. Mai 2016 6:36:14 vorm. schrieb "T. Kim Nguyen" 
<notifications@github.com>:

**Comment by davilima6** [comment]
@jensens, I see your point but I'd say that even for the editor it's semantically a bit different. It's a discrete concept in previous Plone, hidden in a Edit form subtab and only evident through byline when document has expired. No visuals anywhere for to be effective or to be expired content.

That's been a gray Plone concept for years, leading to lots of misunderstanding. We'd like users to learn: "Private objects are inaccessible until you change workflow state to Publish, however they won't be promoted in listings if Effective Date metadata is set but hasn't happened yet. And if they Expire they'll still be accessible through search and direct links but hidden from regular content listings". That's kind of a lot and will be more evident if we move it closer in UI to the workflow states menu.

Otoh it may be useful to inform anonymous users some content's expired, so we'll need some indicator in content area anyway to provide for this other usecase.

**Comment by tkimnguyen** [comment]
Maybe then to make it be truly a question of state we should have the item transitioned to a new "expired" state at the expiry date/time?

**Comment by hvelarde** [comment]
that's a different issue, @tkimnguyen; open it ;-)

**Comment by jensens** [comment]
This is a complex topic. I wrote for this some time ago 
collective.wfeffectiverange - but having this as an addon is completely ok. 
I don't think it is what we want for core!


Am 11. Mai 2016 1:57:28 vorm. schrieb "T. Kim Nguyen" 
<notifications@github.com>:

**Comment by 3dogMcNeill** [comment]
coming from the content side, what @davilima6 and @jensens said is exactly right and why I don't use expired, we transition to private when a content item's "life" is over. I don't care what the technical reason is, it doesn't make sense to me that 'expired' isn't a state.

**Comment by tkimnguyen** [comment]
@jensens I really like https://github.com/collective/collective.wfeffectiverange - brilliant!
